lib/jellyfish_web/controllers/subscription_controller.ex

```diff
@@ -0,0 +1,44 @@
+defmodule JellyfishWeb.SubscriptionController do
+  use JellyfishWeb, :controller
+  use OpenApiSpex.ControllerSpecs
+
+  alias Jellyfish.Room
+  alias Jellyfish.RoomService
+  alias JellyfishWeb.ApiSpec
+  alias OpenApiSpex.Response
+
+  action_fallback JellyfishWeb.FallbackController
+
+  tags [:hls]
+
+  operation :create,
+    operation_id: "subscribe_tracks",
+    summary: "Subscribe hls component for tracks",
```

Comment:
I'm 95% sure it's "subscribe to", not "subscribe for", but you should double-check

Reply:
@mickel8 told me that in this case it's "subscribe for" but I'll double check it.

Reply:
"‘subscribe to’ is related to the service you join and ‘subscribe for’ describes what you receive with your subscription" for example "subscription for bonds or shares"

Reply:
![](https://media.tenor.com/ERa47gPVan8AAAAC/the-more-you-know.gif)